### PR TITLE
Removed obsolete fields from openedx.yaml files

### PR DESCRIPTION
**Pull request by UsamaSadiq**
**Issue:** [BOM-2009](https://openedx.atlassian.net/browse/BOM-2009)

### Description
- Removed obsolete fields from `openedx.yaml` files.
- ref: https://github.com/edx/open-edx-proposals/pull/157

### CODE REVIEW

File: cookiecutter-xblock/openedx.yaml [hunk 1]
```diff
@@ -4,6 +4,3 @@
 oeps:
     oep-2: true
     oep-7: false
-owner: cpennington
-tags:
```
Comment (mraarif) [inline cookiecutter-xblock/openedx.yaml:8]: `tags` shouldn't be removed

Comment (UsamaSadiq) [reply cookiecutter-xblock/openedx.yaml:8]: reverted the removed `tags` field back.

**Review by mzulqarnain1 — APPROVED** [review]

File: python-template/{{cookiecutter.placeholder_repo_name}}/openedx.yaml [hunk 2]
```diff
@@ -1,24 +1,20 @@
 # openedx.yaml
```
Comment (nedbat) [inline python-template/{{cookiecutter.placeholder_repo_name}}/openedx.yaml:1]: While we're here, can we add this comment to the top?
```
# This file describes this Open edX repo, as described in OEP-2:
# https://open-edx-proposals.readthedocs.io/en/latest/oep-0002-bp-repo-metadata.html#specification
```

Comment (UsamaSadiq) [reply python-template/{{cookiecutter.placeholder_repo_name}}/openedx.yaml]: Added the description as per suggestion.

**Comment by nedbat** [comment]
It looks to me like the openedx.yaml file was the only place we used owner_name and owner_type.  This repo still prompts for those values and has plumbing to make them available to the cookiecutter.  @jinder1s can we get rid of that infrastructure?

**Comment by mraarif** [comment]
@UsamaSadiq you can test these cookiecutters locally by creating apps against them, try to locate and remove those prompts for obsolete fields

**Comment by UsamaSadiq** [comment]
Sure @mraarif, I'll create the apps locally and remove the prompts for obsolete fields.

**Review by mraarif — APPROVED** [review]
looks good

**Review by jmbowman — APPROVED** [review]
Nice, that also reduces the number of things slowing down/complicating creation of new repositories.

**Pull request merged**